packages/playground/src/components/ssh_keys/SshDataDialog.vue

```diff
@@ -95,16 +99,39 @@ export default defineComponent({
   setup(props, ctx) {
     const currentKey = ref<SSHKeyData>(props.selectedKey);
     const loading = ref<boolean>(false);
-
+    const hasChanges = ref<boolean>(true);
+    const originalKey = ref<SSHKeyData>({ ...props.selectedKey });
+    const showTooltip = computed(() => !hasChanges.value);
     watch(
       () => props.open,
       newValue => {
         if (newValue) {
           currentKey.value = { ...props.selectedKey };
+          originalKey.value = { ...props.selectedKey };
           loading.value = false;
         }
       },
     );
+    // watch currentKey to detect changes
+    watch(
+      () => currentKey.value,
+      (newValue, oldValue) => {
```

Comment:
``` suggestion
      (newValue) => {
        hasChanges.value = 
          newValue.name !== originalKey.value.name ||
          newValue.publicKey !== originalKey.value.publicKey;
}
```

Reply:
Checking oldValue is useless, especially the `fingerprint` as it's readonly prop